portal/templates/contract/components/ukef-comments.njk

```diff
@@ -1,27 +0,0 @@
-{% macro render(dealStatus, ukefComments, specialConditions) %}
-
-  {% if dealStatus === "Rejected by UKEF" %}
-
-    {% if ukefComments and ukefComments.length %}
-      <p class="govuk-body govuk-!-margin-bottom-2">
-        <span class="govuk-!-font-size-19 govuk-!-font-weight-bold"  data-cy="ukef-comments-title">Reason for rejection:</span> <span class="comment-wrap" data-cy="ukef-comments-text">{{ ukefComments[ukefComments.length - 1].text}}</span>
-      </p>
-    {% endif %}
-
-    {% elif dealStatus !== "Accepted by UKEF (without conditions)" %}
-      
-      {% if dealStatus !== "Accepted by UKEF (with conditions)" and ukefComments and ukefComments.length %}
-        <p class="govuk-body govuk-!-margin-bottom-2">
-          <span class="govuk-!-font-size-19 govuk-!-font-weight-bold"  data-cy="ukef-comments-title">UKEF Comments:</span> <span class="comment-wrap" data-cy="ukef-comments-text">{{ ukefComments[ukefComments.length - 1].text}}</span>
-        </p>
-      {% endif %}
-
-      {% if specialConditions and specialConditions.length %}
-        <p class="govuk-body govuk-!-margin-bottom-2">
-          <span class="govuk-!-font-size-19 govuk-!-font-weight-bold" data-cy="special-conditions-title">Special Conditions:</span> <span class="comment-wrap" data-cy="special-conditions-text">{{ specialConditions[specialConditions.length - 1].text}}</span>
-        </p>
```

Comment:
We should only remove the `specialConditions`part, still need to display other comments